server/utils/generateToken.ts

```diff
@@ -1,9 +1,9 @@
 import { sign } from 'jsonwebtoken';
-import { payloadType } from '../middlewares/interfaces';
+import { payloadType, adminType } from '../middlewares/interfaces';
 
 const key = process.env.SECRET_KEY || '';
 
-const generateToken = (payload:payloadType) => new Promise((resolve, reject) => {
+const generateToken = (payload:payloadType|adminType) => new Promise((resolve, reject) => {
```

Comment:
Why you are adding a new interface, isn't it the same as `payloadType`?

Reply:
yes, it's not the same